scripts/frps_tls.ini

```diff
@@ -0,0 +1,13 @@
+[common]
+log_level = trace
+tcp_mux = true
+bind_port = 7000
+dashboard_port = 7001
+vhost_http_port = 80
+subdomain_host = gradio.live
+detailed_errors_to_client = false
+custom_404_page = /etc/frp/404.html
+
+tls_enable = true
```

Comment:
Updates the FRPS configuration file to include private key and certificate files (these should remain private)